optimade/models/references.py

```diff
@@ -1,22 +1,41 @@
 # pylint: disable=line-too-long,no-self-argument
 from pydantic import (  # pylint: disable=no-name-in-module
-    Field,
     BaseModel,
     AnyUrl,
     validator,
 )
 from typing import List, Optional
 
 from optimade.models.entries import EntryResource, EntryResourceAttributes
+from optimade.models.utils import OptimadeField, SupportLevel
 
 
 __all__ = ("Person", "ReferenceResourceAttributes", "ReferenceResource")
 
 
 class Person(BaseModel):
-    name: str = Field(..., decsription="""Full name of the person, REQUIRED.""")
-    firstname: Optional[str] = Field(None, description="""First name of the person.""")
-    lastname: Optional[str] = Field(None, description="""Last name of the person.""")
+    """A person, i.e., an author, editor or other."""
+
+    name: str = OptimadeField(
+        ...,
+        description="""Full name of the person, REQUIRED.""",
+        support=SupportLevel.OPTIONAL,
```

Comment:
It says in the description this is `REQUIRED`, but this is not reflected in the `support` value.
```suggestion
        description="""Full name of the person, REQUIRED.""",
        support=SupportLevel.MUST,
```